gradio/utils.py

```diff
@@ -292,7 +295,7 @@ def format_ner_list(input_string: str, ner_groups: Dict[str : str | int]):
     return output
 
 
-def delete_none(_dict, skip_value=False):
+def delete_none(_dict: T, skip_value: bool = False) -> T:
```

Comment:
Isn't this type too permissive?

Reply:
Yeah it is technically but I need a way of saying that the output type is the exact same as the input type (otherwise, type checker complains elsewhere). I couldn't find a way of saying that *and* that the type can be one of List, Dict, etc. If you have other suggestions, lmk. Otherwise, will revisit when I come to `utils.py`